### DISCUSSION TITLE
Student developer package, Exchange student, Location problems

### DISCUSSION THREAD

**Discussion by ymb287** [community/community#69475] [GitHub Education]
### Hi everyone,

Hey everyone,

I am trying to sign up for the Student Developer Package but I get the error that I am not close to my campus. This is true since I am on exchange in another country.
But this is blocking my from finishing my application. I can also not upload documents but onyl take pictures wich leads to the problem that I can only make a photo of my Student ID and not the note saying I am on exchange. Did someone experience the same problem? How can I fix that?

Thank you!

**Comment by himanshu-03** [comment]
> ### Hi everyone,
> Hey everyone,
> 
> I am trying to sign up for the Student Developer Package but I get the error that I am not close to my campus. This is true since I am on exchange in another country. But this is blocking my from finishing my application. I can also not upload documents but onyl take pictures wich leads to the problem that I can only make a photo of my Student ID and not the note saying I am on exchange. Did someone experience the same problem? How can I fix that?
> 
> Thank you!
> 
> ### A suggestion...(optional)
> _No response_

Your school-provided academic affiliation document must indicate virtual learning if you are remote. If you are a distance learner then your school-provided academic affiliation documentation must state so.
